### ISSUE TITLE
TV not showup on homekit

### ISSUE THREAD

**Issue by Mashibb** [louis49/homebridge-plugin-androidtv#38]
I did installed the plugins on homebridge. it show on accessory. connect to port 8182 also works. but add it to homekit. it didn't show any devices on the bridge.

**Comment by louis49** [comment]
On port 8182 dis you see and pair a device ?

**Comment by ghost** [comment]
Having this same issue, I paired the device and it works just fine on Homebridge dashboard but the TV doesn't appear in HomeKit at all.

**Comment by BirdFlu101201** [comment]
I am having this issue as well. It is asking me for a pairing code when I go to add it in HomeKit. I’m using a TCL TV

**Comment by cojarbi** [comment]
Scan the Hombridge QR when asked for the code

**Comment by ahmadnazirafiq** [comment]
Mine doesn't even show up on Homebridge nor HomeKit. Here's the logs I see:

`[02/11/2022,` 22:19:42] [HomebridgeAndroidTV] `didFinishLaunching`
`[02/11/2022,` 22:19:42] [HomebridgeAndroidTV] end `didFinishLaunching`
`Setup` Payload: `X-HM://0023N1LTIA4C5`
`[02/11/2022,` 22:19:47] [HomebridgeAndroidTV] Admin server is running on port `8182`
`[02/11/2022,` 22:19:47] [HomebridgeAndroidTV] Finding online device :  Haier TV 192.168.1.133 6466`

It found the TV but I have no idea how to pair it. When tapping + on Home app, it does not show the TV at all. Help!

**Comment by ghost** [comment]
_> 
> It found the TV but I have no idea how to pair it. When tapping + on Home app, it does not show the TV at all. Help!_

Have you tried pairing at `[homebridge_ip]:8060` ?

**Comment by ahmadnazirafiq** [comment]
> _>
> 
> > It found the TV but I have no idea how to pair it. When tapping + on Home app, it does not show the TV at all. Help!_
> 
> Have you tried pairing at `[homebridge_ip]:8060` ?

Yes, but it shows the URL isn't valid. Tried on Safari on either Mac and iPhone, same issue.

**Comment by ahmadnazirafiq** [comment]
The instruction wasn't very clear. Is the port ending homebridge_ip is the homebridge instance port? Or was it something else?

**Comment by nikoloiz** [comment]
> The instruction wasn't very clear. Is the port ending homebridge_ip is the homebridge instance port? Or was it something else?

Hi. 

The web interface for this homebridge plugin is at port 8060. There you can pair a TV,
sorry if my instruction was unclear.

**Comment by ghost** [comment]
> Yes, but it shows the URL isn't valid. Tried on Safari on either Mac and iPhone, same issue.

Hi.
I'm very sorry but gave you the wrong URL. The actual (default) web interface is `[homebridge_ip]:8182` .

**Comment by chenks** [comment]
same issue here
device is correctly paired in homebridge and shows up as an accessory in homebridge,
add the bridge in homekit, the bridge gets added but no device shows up in homekit to control.

**Comment by chenks** [comment]
> Having this same issue, I paired the device and it works just fine on Homebridge dashboard but the TV doesn't appear in HomeKit at all.

did you manage to get this working?

**Comment by Treviko** [comment]
I had to add the accessory manually, did you try that? My bridge has been connected already but the device would not pop up automatically in HomeKit.

**Comment by chenks** [comment]
> I had to add the accessory manually, did you try that? My bridge has been connected already but the device would not pop up automatically in HomeKit.

How do you add it manually?
Any other item in homebridge has always shown up automatically once the bridge has been added

**Comment by Treviko** [comment]
Ok, here is what I did:

- First installation of the plugin (my home bridge is already connected to HomeKit)
-Apple Home App -> Add accessory -> more options -> then I found my tv device

After that I did some minor changes which caused the buttons to stop working. I tried to revert my changes and finally deleted my TV accessory in HomeKit manually.

That's when I started running into the same problem. Config was working and accessory shown in the homebridge. But I have not been able to add or find it via HomeKit.

Now I had two options which you can try:

And this was working for me: In your Homebridge settings you can reset bridges, cameras and tvs. I deleted every old entry, restarted homebridge and after that I could add the accessory manually again via HomeKit.

or
Try to run the plugin as child bridge, add this bridge and test again
